grail/steps.py

```diff
@@ -142,7 +142,7 @@ def print_to_console():
 
 
 def step(func=None, description='', pending=False, step_group=False, format_description=False,
-         treat_nested_steps_as_methods=False, log_output=True, log_input=True):
+         treat_nested_steps_as_methods=False, log_output=True, log_input=True, clean_params=True):
```

Comment:
It depends.
I think it's not so necessary to create a global switch, cos there might be situations when someone will need different types of formatting within one case/class/whatever.
And by default it will stick to the current behaviour.

So, is this switch really needed?